client/gui/translations/ws_desktop_de.ts

```diff
@@ -59,7 +59,7 @@
     </message>
     <message>
         <source> transferred</source>
-        <translation> überführt</translation>
+        <translation> übertragen</translation>
```

Comment:
"Überführt" is something you do in transport like boxes , prisoners and more.
"Übertragen" is something you do with data , money in digital sense and more its more fitting and correct here.